lib/internal/modules/esm/resolve.js

```diff
@@ -1087,6 +1063,40 @@ function defaultResolve(specifier, context = {}) {
     // Ignore exception
   }
 
+  // Import maps are processed first so that policies can still apply
```

Comment:
```suggestion
  // Import maps are processed before policies so policies operate on the final result (precluding any slight-of-hand switch)
```

line-length will need adjustment

Reply:
```suggestion
  // Import maps are processed first so that policies can apply to the result of any mapping
  // defined by the import map.
```

Reply:
I think we should not say "first": they are not the first thing processed (they're just before policies).